### PR TITLE
USDS onboarding another fix - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol [hunk 1]
```diff
@@ -8,29 +8,29 @@ import {IAaveV3ConfigEngine} from 'aave-v3-periphery/contracts/v3-config-engine/
 import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
 import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
 /**
- * @title Onboard USDS and SUSDS
+ * @title Onboard USDS and sUSDS
```
Comment (sendra) [inline src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol:11]:
```suggestion
 * @title Onboard USDS
```

File: src/20240914_Multi_OnboardUSDSAndSUSDS/OnboardUSDSAndSUSDS.md [hunk 2]
```diff
@@ -62,11 +61,11 @@ Following discussions with Sky teams and Aave DAO service providers, the followi
...
 
-- Reserve Factor: Increased from 25% to 50%
-- Variable Rate Slope 1: Decreased from 5.5% to 2%
-- Variable Rate Slope 2: Decreased from 75% to 50%
+- Reserve Factor: Decreased from 25% to 10%
+- Base Variable Borrow Rate: Increased from 0% to 0.75%
+- Variable Rate Slope 1: Increased from 5.5% to 6.25%
 - Optimal Usage Ratio: Increased from 90% to 92%
 
 ## References
```
Comment (brotherlymite) [inline src/20240914_Multi_OnboardUSDSAndSUSDS/OnboardUSDSAndSUSDS.md:71]: Please change the commit hash `efae763a23aefa9fc4506cca5223bcf5a563fea8`, to `main`

File: src/20240914_Multi_OnboardUSDSAndSUSDS/OnboardUSDSAndSUSDS.md [hunk 3]
```diff
@@ -1,25 +1,25 @@
 ---
-title: "Onboard USDS and SUSDS"
+title: "Onboard USDS "
```
Comment (brotherlymite) [inline src/20240914_Multi_OnboardUSDSAndSUSDS/OnboardUSDSAndSUSDS.md:2]:
```suggestion
title: "Onboard USDS on Aave V3 Ethereum Main and Lido Instance"
```

**Review by sendra — APPROVED** [review]

File: src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol [hunk 4]
```diff
@@ -40,16 +40,16 @@ contract AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914 is AaveV3PayloadEthereum {
       ltv: 75_00,
       liqThreshold: 78_00,
```
Comment (sendra) [inline src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol:41]:
```suggestion
      liqThreshold: 72_00,
```

File: src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol [hunk 5]
```diff
@@ -40,16 +40,16 @@ contract AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914 is AaveV3PayloadEthereum {
...
       liqBonus: 7_50,
```
Comment (sendra) [inline src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol:42]:
```suggestion
      liqBonus: 5_00,
```

File: src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol [hunk 6]
```diff
@@ -40,16 +40,16 @@ contract AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914 is AaveV3PayloadEthereum {
...
+      supplyCap: 50_000_000,
+      borrowCap: 45_000_000,
       debtCeiling: 0,
       liqProtocolFee: 10_00,
       rateStrategyParams: IAaveV3ConfigEngine.InterestRateInputData({
         optimalUsageRatio: 92_00,
-        baseVariableBorrowRate: 0,
-        variableRateSlope1: 2_00,
-        variableRateSlope2: 50_00
+        baseVariableBorrowRate: 75,
```
Comment (sendra) [inline src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol:50]:
```suggestion
        baseVariableBorrowRate: 0,
```

File: src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol [hunk 7]
```diff
@@ -40,16 +40,16 @@ contract AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914 is AaveV3PayloadEthereum {
...
+        variableRateSlope1: 6_25,
```
Comment (sendra) [inline src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol:51]:
```suggestion
        variableRateSlope1: 5_50,
```

File: src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol [hunk 8]
```diff
@@ -40,16 +40,16 @@ contract AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914 is AaveV3PayloadEthereum {
...
-      reserveFactor: 50_00,
-      supplyCap: 35_000_000,
-      borrowCap: 32_500_000,
+      reserveFactor: 10_00,
+      supplyCap: 50_000_000,
+      borrowCap: 45_000_000,
       debtCeiling: 0,
       liqProtocolFee: 10_00,
       rateStrategyParams: IAaveV3ConfigEngine.InterestRateInputData({
         optimalUsageRatio: 92_00,
```
Comment (sendra) [inline src/20240914_Multi_OnboardUSDSAndSUSDS/AaveV3Ethereum_OnboardUSDSAndSUSDS_20240914.sol:49]:
```suggestion
        optimalUsageRatio: 90_00,
```

**Review by sendra — APPROVED** [review]

**Review by brotherlymite — APPROVED** [review]

**Pull request merged**